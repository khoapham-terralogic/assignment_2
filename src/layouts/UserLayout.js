import React from 'react';
import UserPage from '../pages/User/UserPageConnectWrapper';

const UserLayout = () => {

    return (
        <div className="user__container">
            <div className="row user__container__header">
                <div className="col-xs-12 user__container__header__wrapper">
                    <div className="user__container__header__title">
                        My Profile
                    </div>
                    <div className="user__container__header__description">
                        Manage your profile and contact information.
                    </div>
                </div>
            </div>
            <UserPage />
        </div>
    );
}

export default UserLayout;
