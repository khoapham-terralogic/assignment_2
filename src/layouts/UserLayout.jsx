import React from 'react';
import UserPage from '../features/auth-feature/main/pages/UserPage';

const UserLayout = () => {

    return (
        <div className="user-container">
            <div className="row user-container-header">
                <div className="col-xs-12 user-container-header-wrapper">
                    <div className="user-container-header-title">
                        My Profile
                    </div>
                    <div className="user-container-header-description">
                        Manage your profile and contact information.
                    </div>
                </div>
            </div>
            <UserPage />
        </div>
    );
}

export default UserLayout;
