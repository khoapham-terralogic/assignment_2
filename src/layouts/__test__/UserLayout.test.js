import React from 'react'
import UserLayout from '../UserLayout'
import { shallow } from 'enzyme';

describe('Test <UserLayout />', () => {
    it('should render', () => {
        const wrapper = shallow(<UserLayout />)
        expect(wrapper).toMatchSnapshot()
    });
    it('should render correctly', () => {
        const wrapper = shallow(<UserLayout />)
        expect(wrapper.find(".user__container__header__title").text()).toEqual("My Profile")
        expect(wrapper).toMatchSnapshot()
    });
})
