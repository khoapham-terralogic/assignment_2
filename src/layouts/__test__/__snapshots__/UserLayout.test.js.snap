// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <UserLayout /> should render 1`] = `
<div
  className="user__container"
>
  <div
    className="row user__container__header"
  >
    <div
      className="col-xs-12 user__container__header__wrapper"
    >
      <div
        className="user__container__header__title"
      >
        My Profile
      </div>
      <div
        className="user__container__header__description"
      >
        Manage your profile and contact information.
      </div>
    </div>
  </div>
  <Connect(UserPage) />
</div>
`;

exports[`Test <UserLayout /> should render correctly 1`] = `
<div
  className="user__container"
>
  <div
    className="row user__container__header"
  >
    <div
      className="col-xs-12 user__container__header__wrapper"
    >
      <div
        className="user__container__header__title"
      >
        My Profile
      </div>
      <div
        className="user__container__header__description"
      >
        Manage your profile and contact information.
      </div>
    </div>
  </div>
  <Connect(UserPage) />
</div>
`;
